lib/active_record/connection_adapters/sqlserver_adapter.rb

```diff
@@ -194,7 +196,7 @@ def reset!
 
       def tables_with_referential_integrity
         schemas_and_tables = select_rows <<-SQL.strip_heredoc
-          SELECT s.name, o.name
+          SELECT s.name sc, o.name tab
```

Comment:
Another nice change. Did JDBC throw a warning here or did you notice the ambiguity?